k8s/helm/yona-server/values.yaml

```diff
@@ -166,10 +134,12 @@ sms:
   
 apple:
   enabled: false
+  keystore_secret_filename: dummy.p12
```

Comment:
It was just something different between prod and default that I had to alter on that day.

The filename is not dummy on prod, but I didn't want to keep default values to generate as close to the same reference material in resources/*

Making it a variable allowed me to set the production value to the desired production name.  We can make it all the same as on production, just need to change values.yaml.    I have not altered the PR to reflect this though.